tests/testutil/config.go

```diff
@@ -114,3 +126,18 @@ func (tc *TestConfig) waitForConfig() error {
 	}
 	return nil
 }
+
+// GetContent returns the configs Data.Content as list of strings, newline separated
+func (tc *TestConfig) getContentData(cm corev1.ConfigMap) []string {
```

Comment:
Should probably keep this as standalone method, off the TestConfig struct, to avoid confusion.